config/config.go

```diff
@@ -279,11 +279,16 @@ type RemoteAuth struct {
 	SignalingCreds         *rpc.Credentials `json:""`
 }
 
+var remoteNameRegEx = regexp.MustCompile(`^[A-Za-z0-9_-]+$`)
```

Comment:
should we consolidate these name checking regexes?

Reply:
yeah we should consolidate - note that the other validator we have enforces resource names to begin with a letter, though that shouldn't really affect your change since it's a stricter check.